_posts/2016-03-16-edition-13.markdown

```diff
@@ -298,7 +298,7 @@ __Light reading__
 * [Git rebase and the golden rule explained](https://medium.freecodecamp.com/git-rebase-and-the-golden-rule-explained-70715eccc372#.93ok9bhsw), by Pierre de Wulf
 * [GitHub Workflow (used by Frameworks team at BBC News)](http://www.integralist.co.uk/posts/github-workflow.html)
 * [Write, Review, Merge, Publish: Phabricator Review Workflow](https://secure.phabricator.com/phame/post/view/766/write_review_merge_publish_phabricator_review_workflow/)
-* [A succesful Git branching model considered harmful](https://barro.github.io/2016/02/a-succesful-git-branching-model-considered-harmful/), by Jussi Judin
+* [A successful Git branching model considered harmful](https://barro.github.io/2016/02/a-successful-git-branching-model-considered-harmful/), by Jussi Judin
```

Comment:
The change in the URL breaks the link here. Even if it's a typo, we shouldn't make the change in the URL.

Reply:
Also the title of the linked article has the typo too, so not sure we actually should make the change in the title too.

Reply:
Absolutely, let's keep it as is, thanks!